5thColumn/vee
# Fetch Interrogated task reports

Added an endpoint to retrieve reports from
Interrogated. Since the reports are generally
huge, this includes filesystem caching.

This commit also servers as the first commit,
and therefore contains standard node.js files.

TO TEST:
You need a valid report ID to hit. I used 393. Others can be found in interrogated as necessary. Make sure to hit the same report twice to make sure there's no issues in the filesys caching.
